### PR TITLE
Add SSL verification options

### PR DESCRIPTION
**Pull request by combor**
- add CA bundle parameter
  This allows to provide a CA that should be used to verify s3 endpoint certificate
- Add skip_ssl_verification option
  This add an option to skip SSL verification while connecting to an
    endpoint.

### CODE REVIEW

File: assets/check [hunk 1]
```diff
@@ -15,6 +15,8 @@ suffix=$(echo "${payload}" | jq -r '.source.suffix // ""')
 endpoint_url="$(echo "$payload" | jq -r '.source.endpoint // ""')"
 region="$(echo "$payload" | jq -r '.source.region // "us-east-1"')"
 use_v4="$(echo "$payload" | jq -r '.source.use_v4 // ""')"
+skip_ssl_verification="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
+ca_bundle="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
```
Comment (jimconner) [inline assets/check]: We're setting ca_bundle here, but we're fetching the skip_ssl_validation field from the payload. Is that correct? It looks like a cut 'n' paste that didn't get edited.

File: assets/in [hunk 2]
```diff
@@ -23,6 +23,8 @@ use_v4="$(echo "${payload}" | jq -r '.source.use_v4 // ""')"
 path=$(echo "${payload}" | jq -r '.source.path // ""')
 suffix=$(echo "${payload}" | jq -r '.source.suffix // ""')
 encryption_key="$(echo "${payload}" | jq -r '.source.encryption_key // ""')"
+skip_ssl_verification="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
+ca_bundle="$(echo "$payload" | jq -r '.source.skip_ssl_verification // ""')"
```
Comment (jimconner) [inline assets/in]: Same here.. ca_bundle is being set as source.skip_ssl_validation.

File: assets/check (hunk 1 above)
Comment (combor) [reply assets/check]: Fixed.

File: assets/in (hunk 2 above)
Comment (combor) [reply assets/in]: Fixed.

**Review by jimconner — APPROVED** [review]

**Pull request merged**